.github/workflows/ci.yml

```diff
@@ -1,83 +1,101 @@
 name: CI
 
 on:
+  schedule:
+    - cron: "00 19 * * *" # run ci periodically at 3 am
   pull_request:
     branches: [master]
 
 env:
   CI_RUST_TOOLCHAIN: 1.61.0
 
 jobs:
-  outdated:
```

Comment:
Why do you comment out all these jobs?

Reply:
Work In Progress. Had to open pr for the ci to run. Have reverted now